local-proxy/src/server.ts

```diff
@@ -14,11 +14,13 @@ import { AddressInfo } from 'net';
 import os from 'os';
 import { initRegistry } from './stdio';
 import nanoid from 'nanoid';
-import mkdirp from 'mkdirp';
-import { copy } from 'fs-extra';
+import { copy, mkdirpSync } from 'fs-extra';
 import env from 'env-var';
 import dotenv from 'dotenv';
 
+const tmpDir = env.get('RESHUFFLE_TMP_DIR').required().asString();
```

Comment:
Can still be checked for `isAbsolute()`, just in case.

Reply:
Does it matter?

Reply:
If it's e.g. `./.reshuffle` and the code does `process.chdir()` somewhere it suddenly breaks.

Reply:
It's not possible ATM since we use `tmpDir` with `resolvePath` before requiring user code.
We might want to `resolvePath` immediately for `tmpDir` to prevent future errors.

Reply:
It's not possible ATM, but so are all other environment variable checks but it comes to protect from 99% of mistakes where an `undefined` or empty string are passed.